eng/Dependencies.props

```diff
@@ -164,6 +164,7 @@ and are generated based on the last package release.
     <LatestPackageReference Include="MessagePack" Version="$(MessagePackPackageVersion)" />
     <LatestPackageReference Include="Mono.Cecil" Version="$(MonoCecilPackageVersion)" />
     <LatestPackageReference Include="Moq" Version="$(MoqPackageVersion)" />
+    <LatestPackageReference Include="NETStandard.Library" Version="2.0.3" />
```

Comment:
You need to reference this in the failing test projects somehow

Reply:
I actually have a fix, can you hold off pushing?

Reply:
Oh, I just forgot to ctrl-s my other changes before pushing 😆

Reply:
I'll defer to your change @ericstj